### PR TITLE
Version next treegridviewdemo

### PR DESCRIPTION
**Pull request by Wagnerp**
* Some changes
* Please review 'TODO' comment in `Krypton.Toolkit.Suite.Extended.DataGridView 2019.csproj`, as this is the output when running `build.cmd`

image

### CODE REVIEW

File: Source/Krypton Toolkit/Third Party/Microsoft.WindowsAPICodePack.Core/AppRestartRecovery/ApplicationRecoveryException.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+﻿namespace Microsoft.WindowsAPICodePack.Core
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Third Party/Microsoft.WindowsAPICodePack.Core/AppRestartRecovery/ApplicationRecoveryException.cs]: Why not pull this in as a nuget pack ?
Have you done specific fixes that are not included in V 1.1.1 -> 1.1.4?

image

File: Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.DataGridView/Krypton.Toolkit.Suite.Extended.DataGridView 2019.csproj [hunk 2]
```diff
@@ -52,6 +52,7 @@
         <OutputPath>..\..\..\..\Bin\$(configuration)\Main\Krypton.Toolkit.Suite.Extended.DataGridView\</OutputPath>
     </PropertyGroup>
   
+  <!--TODO: Please fix, as this causes the compiler to go crazy-->
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.DataGridView/Krypton.Toolkit.Suite.Extended.DataGridView 2019.csproj:55]: Probably need matching against framework <= .net 4.8

File: Source/Krypton Toolkit/Third Party/Scintilla.NET/Scintilla.NET.csproj [hunk 3]
```diff
@@ -1,8 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Choose>
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Third Party/Scintilla.NET/Scintilla.NET.csproj:3]: This should be nuget ted !

File: Source/Krypton Toolkit/Third Party/Simple.MAPI.NET/Simple.MAPI.NET.csproj [hunk 4]
```diff
@@ -1,8 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
+  <Choose>
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Third Party/Simple.MAPI.NET/Simple.MAPI.NET.csproj:3]: This should be nuget`ted

File: Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Dialogs/UX/Internal/InternalExtendedKryptonForm.cs [hunk 5]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Krypton.Toolkit.Suite.Extended.Dialogs
+{
+    /// <summary>DO NOT USE!!!!!!!!! The purpouse of this is to expand the capabilities of a normal <seealso cref="KryptonForm"/>.</summary>
+    public class InternalExtendedKryptonForm : KryptonForm
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Dialogs/UX/Internal/InternalExtendedKryptonForm.cs:10]: Make it internal, then set the class dll's that extend it to be "Friends" of this dll.

File: Source/Krypton Toolkit/Third Party/Microsoft.WindowsAPICodePack.Core/Class1.cs [hunk 6]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Third Party/Microsoft.WindowsAPICodePack.Core/Class1.cs:1]: Delete this ?

File: Source/Krypton Toolkit/Third Party/Microsoft.WindowsAPICodePack.Core/AppRestartRecovery/ApplicationRecoveryException.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Third Party/Microsoft.WindowsAPICodePack.Core/AppRestartRecovery/ApplicationRecoveryException.cs]: No, just made the license agreement (MIT) mandatory, though it doesn't include the fixes contained in this repo: https://github.com/ljvankuiken/WindowsAPICodePack

File: Source/Krypton Toolkit/Third Party/Scintilla.NET/Scintilla.NET.csproj (hunk 3 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Third Party/Scintilla.NET/Scintilla.NET.csproj:3]: Would fix a lot of compiler errors! :)

**Pull request merged**